voter-dapp/src/ActiveRequests.js

```diff
@@ -367,7 +371,7 @@ function ActiveRequests({ votingAccount, votingGateway }) {
       ) : (
         ""
       )}
-      {!canExecuteBatch() ? (
+      {(saveButtonShown && !canExecuteBatchCommit()) || (revealButtonShown && !canExecuteBatchReveal()) ? (
         <span style={{ paddingLeft: "10px", color: "#FF4F4D" }}>
           You can only commit or reveal up to 25 requests at once. Please select fewer.
```

Comment:
Parameterize the 25 in this text as well.

Reply:
done